doc/sphinxman/source/prog_faq.rst

```diff
@@ -89,7 +89,7 @@ Workflow
 Miscellaneous
 -------------
 
-#. :ref:`faq:add_tests`
+#. :ref:`faq:add_psithon_tests`
```

Comment:
Can we still link `add_tests` but also link the `add_psithon_tests` and `add_psiapi_tests` as sub-bullets?

Reply:
Added back the general link. The psithon and psiapi are sub-subbullets in the TOC. The faq is intended to expand on available answers that might not be apparent from the TOC overview. It's not too thorough for the "programmers manual" at the moment.